### PR TITLE
Replace `master` with `main` where possible

### LINKED ISSUE #1204: Should `main` branch be used in install docs?
Hi, I was wondering if the `main` branch should be used [here](https://helm.sh/docs/intro/install/#from-script) instead of the `master` branch?

### PR DESCRIPTION
**Pull request by bertdeblock**
This replaces most of the `master` occurrences with `main`. Changes in URLs have been checked manually. Other changes weren't super straight forward, specially for the languages I don't speak, so please let me know if some changes aren't correct and I'll update accordingly.

Closes #1204.

Signed-off-by: Bert De Block <bert.deblock@bagaar.be>

### CODE REVIEW

File: content/code/helm-v3.md [hunk 1]
```diff
@@ -2,5 +2,5 @@
 url: "helm/v3"
 name: "helm/v3"
 repoURL: "https://github.com/helm/helm"
-branch: "master"
+branch: "main"
```
Comment (bertdeblock) [inline content/code/helm-v3.md:5]: Not sure if these changes are okay?

**Comment by bertdeblock** [comment]
Should `**master branch**` be changed here as well?
https://github.com/helm/helm-www/blame/main/content/en/docs/topics/chart_repository.md#L186

**Comment by bacongobbler** [comment]
> Should `**master branch**` be changed here as well?

You could change it to read "your default branch" instead.

File: content/code/helm-v3.md (hunk 1 above)
Comment (bacongobbler) [reply content/code/helm-v3.md:5]: they're fine

**Review by bacongobbler — APPROVED** [review]

**Review by bacongobbler — APPROVED** [review]

**Pull request merged**